.circleci/config.yml

```
@@ -36,10 +54,31 @@ jobs:
             - run:
                 name: Override version of eslint@<<parameters.eslint-version>>
                 command: yarn add eslint@<<parameters.eslint-version>> --dev
-      - run: yarn format:check
-      - run: yarn lint
-      - run: yarn test
+      - run: 
+          name: Check formatting
+          command: yarn format:check
+      - run:
+          name: Run linter
+          command: yarn lint
+      - run:
+          name: Run unit tests
+          command: yarn test
 
+  deploy:
+    docker:
+      - image: circleci/node
```

```suggestion
      - image: cimg/node:16.14.0
```

`circleci` is the [legacy](https://discuss.circleci.com/t/legacy-convenience-image-deprecation/41034) images, `cimg` is the new ones.

---

Was thinking to upgrade from legacy to the new images in a separate PR for better separation